util/client/index.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Capitalizes the first letter of a string
+ */
+export const capitalizeFirstLetter = (value: string): string =>
+  value[0].toUpperCase() + value.slice(1);
```

Comment:
shouldnt you check if the  string is not empty? it may throw an exception if its an empty string or a one word string

Reply:
Ill do a check

Reply:
looking further into this an empty string or undefined will never occur